### PR TITLE
docs: Fix formatting of `GCE_METADATA_HOST`

### PR DESCRIPTION
**Pull request by ijrsvt**
The existing docs only show `GCE_METADATA_ROOT`. This PR adds a space between them so that Sphinx will separate them into separate entries.
Screen Shot 2021-10-16 at 9 06 11 AM
The new HTML: 
```
<dl class="data">
<dt id="google.auth.environment_vars.GCE_METADATA_HOST">
<code class="sig-name descname">GCE_METADATA_HOST</code><em class="property"> = 'GCE_METADATA_HOST'</em><a class="headerlink" href="#google.auth.environment_vars.GCE_METADATA_HOST" title="Permalink to this definition">¶</a></dt>
<dd><p>port to be
used for GCE metadata requests.</p>
<p>This environment variable was originally named GCE_METADATA_ROOT. The system will
check this environemnt variable first; should there be no value present,
the system will fall back to the old variable.</p>
<dl class="field-list simple">
<dt class="field-odd">Type</dt>
<dd class="field-odd"><p>Environment variable providing an alternate hostname or host</p>
</dd>
</dl>
</dd></dl>

<dl class="data">
<dt id="google.auth.environment_vars.GCE_METADATA_ROOT">
<code class="sig-name descname">GCE_METADATA_ROOT</code><em class="property"> = 'GCE_METADATA_ROOT'</em><a class="headerlink" href="#google.auth.environment_vars.GCE_METADATA_ROOT" title="Permalink to this definition">¶</a></dt>
<dd><p>Old environment variable for for GCE_METADATA_HOST.</p>
</dd></dl>
```

### CODE REVIEW

File: google/auth/environment_vars.py [hunk 1]
```diff
@@ -41,15 +41,17 @@
...
-This environment variable is originally named GCE_METADATA_ROOT. System will
-check the new variable first; should there be no value present,
-the system falls back to the old variable.
+This environment variable was originally named GCE_METADATA_ROOT. The system will
+check this environemnt variable first; should there be no value present,
+the system will fall back to the old variable.
 """
 
+GCE_METADATA_ROOT = "GCE_METADATA_ROOT"
+"""Old environment variable for for GCE_METADATA_HOST."""
```
Comment (arithmetic1728) [inline google/auth/environment_vars.py:53]:
```suggestion
"""Old environment variable for GCE_METADATA_HOST."""
```

**Review by arithmetic1728 — APPROVED** [review]

**Comment by ijrsvt** [comment]
@arithmetic1728 do you know why `Kokoro` is not running?

**Comment by arithmetic1728** [comment]
@ijrsvt I am not sure. Just triggered kokoro to re-run.

**Comment by ijrsvt** [comment]
@arithmetic1728 It looks like it passes now! Can you add the `automerge` label? Thanks :)

**Pull request merged**